pods-framework/pods
# Set wpautop to true in the TinyMCE settings during set up

Fixes #7049

## Testing instructions

1. Set up a WYSIWYG field
2. Use content with many paragraphs
3. Edit in admin area: Expect the content to retain line breaks in editor
4. Edit on front of site: Expect the content to retain line breaks in editor (previously no line breaks would be treated as paragraphs

## Changelog text for these changes

* Fixed: TinyMCE on frontend would sometimes not show line breaks as paragraphs.

## PR checklist
